syntax = "proto3";

service StandardGrpcService {
    rpc standardInfer(Request) returns (Response) {};
}

message Request {
    string stream_name = 1;
    bytes image_data = 2;
}

message Response {
    int32 code = 1;
    string message = 2;
}